### ISSUE TITLE
People search template update

### ISSUE THREAD

**Issue by pilz** [ploneintranet/ploneintranet#754]
Seems the template of the people search is no longer in sync with proto. I do see name and email but no additional attributes as shown in proto. Also the proto header has a slight gray background, which hints to more changes, even some my untrained eye may not see immediately.

@mattss  interested?

**Comment by mattss** [comment]
Last time I checked, the people search template included Title, email and telephone.

The prototype includes a tag line or 'latest status', which was ditched from the prototype individual profile view but still appears in search results for some reason - that may be a mistake. Either way, there's no backend for this that I'm aware of.

The prototype also includes a list of endorsements, which there is also no backend for and I'm not aware is in the current feature set for venus.

**Comment by gyst** [comment]
Yes 'latest status' should be dropped. In the future will become 'last seen' or something like that but out of scope for Venus.

Endorsements: actually the backend is already implemented (in ploneintranet.network), it's only the frontend implementation that's missing. But out of scope for Venus indeed also.
